decidim-proposals/spec/shared/manage_proposals_permissions_examples.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+shared_examples "manage proposals permissions" do
+  context "when authorization handler is Everyone" do
```

Comment:
Looks like the test descriptions are not correct

Reply:
Sorry, I don't guess why.. would you prefer "when *withdraw* authorization handler is Everyone" ?

Reply:
Ah, I misunderstood the tests, sorry! Texts are correct indeed! 😄